### PR TITLE
Update SIG Network reviewers

### PR DESCRIPTION
**Pull request by shaneutt**
#### What type of PR is this?

/kind cleanup
/kind documentation

#### What this PR does / why we need it:

Our approvers and reviewers file had gotten way out of day, with some inactive users still present and not having reflected the leads changes of the last year.

#### Special notes for your reviewer:

/cc @thockin check this out

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -358,12 +358,12 @@ aliases:
     - neolit123
     - vincepri
   sig-network-api-approvers:
-    - smarterclayton
+    - danwinship
```
Comment (thockin) [inline OWNERS_ALIASES:361]: I think API approvers need to go thru API review "training" (shadowing, mostly).  Happy to do that, of course.

**Review by thockin — COMMENTED** [review]
Otherwise LGTM

**Comment by shaneutt** [comment]
Will come back to this later after we've done shadowing.

**Pull request closed without merging**